src/components/notes/editor.jsx

```diff
@@ -134,6 +126,20 @@ const Editor = translate()(
       [props.returnUrl, doc]
     )
 
+    // Failure in loading the note ?
+    useEffect(
+      () => {
+        if (!loading && !doc) {
+          // eslint-disable-next-line no-console
+          console.warn(`Could not load note ${noteId}`)
+          if (returnUrl) {
+            document.location = returnUrl
+          }
```

Comment:
Should we have an `else` and push to `/` as well?

Reply:
We could change the returnUrl logic above to have '/' as fallback if we are in a private view but '/' is not ok for a public view (it would trigger the login form on a cozy which is not yours)